1. objects/1.md

```diff
@@ -2,72 +2,110 @@
 
 ```js
 // your code goes here
+var myPenguin = {
+    name : "whiteblack",
+    origin : "Whiteblack the Penguin Sees the World",
+    author : "H. A. Rey and Margret Rey",
+    notes : "Children's book about a travelling penguin"
+    }
 ```
 
 2. Use console.log() to print the penguin's name to the console as part of a welcome message, like "Hello, I'm a penguin and my name is [NAME HERE]!"
 
 ```js
 // your code goes here
+console.log("Hello, I'm a penguin and my name is [" + myPenguin.name + "]!");
```

Comment:
It's a string `" + myPenguin.name + "` not a variable you can use template literal.